lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -292,6 +292,18 @@ <h4 class="message-title">
 
 
   </section>
+  % if course_requirements:
+  <% prc_target = reverse('about_course', args=[course_requirements['prc_id'].to_deprecated_string()]) %>
+  <section class="prerequisites">
+    <p class="tip">
+      ${_("You must successfully complete {link_start}{prc_display}{link_end} before you begin this course.").format(
+          link_start='<a href="{}">'.format(prc_target),
+          link_end='</a>',
+          prc_display=' '.join([course_requirements['prc_id'].org, course_requirements['prc_id'].course]),
```

Comment:
Using combination of `display_org_with_default` and `display_number_with_default` as suggested by @marcotuts